examples/alloydb/cluster.yaml

```
@@ -0,0 +1,25 @@
+apiVersion: alloydb.gcp.upbound.io/v1beta1
+kind: Cluster
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alloydb/v1beta1/cluster
+  labels:
+    testing.upbound.io/example-name: default
+  name: default
+spec:
+  forProvider:
+    location: us-central1
+    network: projects/${data.google_project.project.number}/global/networks/${google_compute_network.default.name}
```

```suggestion
    networkSelector:
      matchLabels:
        testing.upbound.io/example-name: default
```

It is better to use a selector so that our examples can be tested automatically via uptest.